front/lib/api/assistant/token_pricing.ts

```diff
@@ -216,9 +216,9 @@ const CURRENT_MODEL_PRICING: Record<BaseModelIdType, PricingEntry> = {
     input: 0.55,
     output: 2.19,
   },
-  "accounts/fireworks/models/deepseek-r1": {
-    input: 8.0,
-    output: 8.0,
+  "accounts/fireworks/models/deepseek-r1-0528": {
```

Comment:
Added a commit to fix pricing, from: https://fireworks.ai/models/fireworks/deepseek-r1-0528